src/Symfony/Component/Workflow/Attribute/AsAnnounceListener.php

```diff
@@ -22,10 +24,29 @@ final class AsAnnounceListener extends AsEventListener
     use BuildEventNameTrait;
 
     public function __construct(
+        /*
+         * The id of the workflow to listen to.
```

Comment:
This need to move back to `@param` 😕

Reply:
I'll do a new pass on this PR when the other PR is all good so I know what to change all at once. Thank you for this early review Jerôme and spotting these typos! 😃